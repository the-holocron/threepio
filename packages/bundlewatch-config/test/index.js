/* eslint-env jest */
const config = require('../bundlewatch.config.json');

describe('Preset', () => {
	describe('Bundlewatch', () => {
		it('should render the configuration', () => expect(config).toMatchSnapshot());
	});
});
