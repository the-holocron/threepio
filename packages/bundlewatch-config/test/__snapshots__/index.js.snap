// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preset Bundlewatch should render the configuration 1`] = `
Object {
  "files": Array [
    Object {
      "maxSize": "100 kB",
      "path": "./dist/**/*.js",
    },
  ],
}
`;
